src/test/java/seedu/address/logic/commands/DependencyCommandTest.java

```diff
@@ -63,20 +69,27 @@ public void execute_cyclicDependencyUnfilteredList_throwsCommandException() {
     }
 
     @Test
-    public void execute_validIndexFilteredList_success() {
+    public void execute_addValidIndexFilteredList_success() {
```

Comment:
The test name is not consistent with the execute_validIndexUnfilteredList_success() test. this test isn't just testing adding dependency rt?

Reply:
Right

Reply:
Done